src/js/view/controls/controlbar.js

```diff
@@ -238,6 +239,9 @@ define([
             _model.change('cues', this.addCues, this);
             _model.change('altText', this.setAltText, this);
 
+            _model.on('addButton', this.addButton, this);
+            _model.on('removeButton', this.removeButton, this);
```

Comment:
These would only get called when controls are active meaning you could not add buttons unless controls are true, loaded and the controlbar created.

The creation of buttons should be separate from the rendering of them in controlbar. (see my previous comment in controller.js).